src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/CountIf.cs

```diff
@@ -76,8 +76,15 @@ public override bool CheckTypes(TexlNode[] args, DType[] argTypes, IErrorContain
                 }
```

Comment:
Do we need this separate check for OptionSetValue in that case?

Reply:
We already have a check for OptionSetValue. This extends it to other types.

Reply:
Ah, good question. Let me try it out